scripts/seed-db-from-files.js

```diff
@@ -106,7 +106,10 @@ const start = async (sourceDirectory, mod, markDirty) => {
   console.group(`Inserting ${courses.length} courses to db...`)
   /** @type {{id: number, name: string}[]} */
   const insertedCourses = await knex
-    .batchInsert('courses', courses.map(({ name }) => ({ name })))
```

Comment:
Would it be too much work to separate ’run prettier’ into own commit so we can https://michaelheap.com/git-ignore-rev/ ?

Reply:
prettier only stuff splitted into own commit